packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -887,14 +904,15 @@ function UpgradeCreditInformation( { product }: { product: ResponseCartProduct }
 		);
 	}
 
-	if ( isYearly( product ) ) {
+	if ( isYearly( product ) && humanReadableCode ) {
 		return (
 			<>
-				{ translate( 'Upgrade Credit: %(upgradeCredit)s applied in first year only', {
+				{ translate( '%(humanReadableCode)s : %(upgradeCredit)s applied in first month only.', {
```

Comment:
`applied in first month only.` should this read as `year` for this block?

Reply:
Great catch, I have resolved it.